modules/administration/pages/public-cloud-azure.adoc

```diff
@@ -0,0 +1,164 @@
+
+When you are setting up disk partitioning, we recommend:
+
+* 30{nbsp}GB for the disk running the operating system
+* Select `Standard HDD` for the storage account type
+
+You will also require three additional data disks:
+
+* Disk 0: 64{nbsp}GB on Premium SSD, mounted at [path]``/var/lib/pgsql``
+* Disk 1: 512{nbsp}GB on Standard SSD, mounted at [path]``/var/spacewalk``
+* Disk 2: 128{nbsp}GB on Standard SSD, mounted at [path]``/var/cache``
+
+[NOTE]
+====
+Do not use LVM with Azure.
+If you need more disk space, extend a disk in the Azure portal, then extend the filesystem with [command]``xfs_growfs``.
+====
+
+Partition the disks like this:
+
+* [path]``/dev/sda``: 4 partitions containing the OS
+* [path]``/dev/sdb``: temporary storage disk, do not use
+* [path]``/dev/sdc``: contains [path]``/var/lib/pgsql``
+* [path]``/dev/sdd``: contains [path]``/var/spacewalk``
+* [path]``/dev/sde``: contains [path]``/var/cache``
+
+You can use these commands to create the disks:
+
+----
+for d in sdc sdd sde; do
+  parted --script /dev/$d mklabel gpt mkpart primary xfs 0% 100%
+  mkfs.xfs /dev/${d}1
+done
+mkdir /cachetmp
+mount /dev/sde1 /cachetmp
+cp -a /var/cache/* /cachetmp/
+umount /cachetmp
+echo "$(blkid /dev/sdc1|awk -F " " '{ print $2 }') /var/lib/pgsql xfs defaults,noatime 0 0" >> /etc/fstab
+echo "$(blkid /dev/sdd1|awk -F " " '{ print $2 }') /var/spacewalk xfs defaults,noatime 0 0" >> /etc/fstab
+echo "$(blkid /dev/sde1|awk -F " " '{ print $2 }') /var/cache     xfs defaults,noatime 0 0" >> /etc/fstab
+mkdir -p /var/spacewalk
+mount /var/spacewalk
+chown -R wwwrun:root /var/spacewalk
+mount /var/lib/pgsql
+chown -R postgres:postgres /var/lib/pgsql
+mv /var/cache /var/cache.old
+mkdir /var/cache
+mount /var/cache
+rm -r /var/cache.old
+----
+
+
+// REMARK: I guess you do this in your Azure instance
+When you are setting up networking, we recommend that you create a separate private network, with the IP range `10.0.0.0/24`.
+// REMARK: Where do you configure this?
+
+Configure the {productname} Server to use the internal IP address `10.0.0.4`.
+Ensure it is accessible from outside the network with IP `10.89.171.101`.
+
+Configure the firewall to only allow inbound traffic on ports `22`, `80`, and `443` to IP address `10.0.0.4`.
+In this environment, if other servers are added to the network they cannot be reached from outside the network.
+
+Outbound is open from the private network.
+// REMARK: Was does this mean?
+This should be restricted for other servers in this private network.
+
+You will need to set the DNS zones in Azure before you can configure the {productname} Server.
+For more information on setting DNS zones, see the Azure documentation.
+
+
+
+== Configure {productname} Server
+
+Ensure that your {productname} Server is registered with {scc}.
+
+When your server is registered, install these extra packages:
+----
+zypper -n in spacecmd mlocate sysstat
+----
```

Comment:
Please note that `spacecmd` has been added to the SUSE Manager Server images (though not released yet). Any particular reason why `mlocate` and `sysstat` are required?

Reply:
That is correct. The document was written on the moment it wasn't. I forgot that also spacewalk-utils is needed (eg for taskotop which is a useful utility to see what taskomatic is doing). 

mlocate is just a tool I often use the search for files. And yes you can also use "find", but like locate and most customer I worked with share that feeling.

Sysstat helps you troubleshooting performance issues or to check the general health. Also this is often requested by support. Still fighting with PM to get this on every SLES as default (for SLES4SAP it is).